cvat/apps/engine/permissions.py

```diff
@@ -423,6 +445,7 @@ class Scopes(StrEnum):
         UPDATE_ASSOCIATED_STORAGE = 'update:associated_storage'
         UPDATE_DESC = 'update:desc'
         UPDATE_METADATA = 'update:metadata'
+        # complex scope consisting of a combination of other scopes
```

Comment:
It's not clear to which scope this comment is related to and it doesn't really add useful information. There are several composite scopes in this class.